### ISSUE TITLE
bug: remove the obsolete "status" field from W3C error responses

### ISSUE THREAD

**Issue by joerg1985** [appium/appium#20321]
### Do I have the most recent component updates?

- [X] I use the most recent available driver/plugin and server versions

### Is the component officially supported by the Appium team?

- [X] I have verified the component repository is present under the Appium organization in GitHub

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Appium 2.10.3 does still use none W3C encoded errors.

E.g. calling http://127.0.0.1:4723/test/ does result in this error:
```
{
    "status": 9,
    "value": {
        "error": "unknown command",
        "message": "The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource",
        "stacktrace": ""
    }
}
```

### Expected Behavior

Appium should use W3C encoded errors.
https://www.w3.org/TR/webdriver2/#example-3

### Minimal Reproducible Example

1. run appium
2. call http://127.0.0.1:4723/test/

### Environment

- Operating system: Windows 10 x64
- Appium server version (output of `appium --version`): 2.10.3
- Appium driver(s) and their version(s) (`appium driver list`):
- Appium plugin(s) and their version(s) (`appium plugin list`):
- Node.js version (output of `node --version`):
- `npm` version (output of `npm --version`):
- Last component(s) version which did _not_ exhibit the problem:
- Platform and version under test:
- Real device or emulator/simulator:

### Further Information

I created a [PR](https://github.com/SeleniumHQ/selenium/pull/14195) some days ago to remove the support of none W3C encoded errors from Selenium. It is totally open whether this gets merged or not.

**Comment by mykola-mokhnach** [comment]
Could you please provide an example of the expected response? 

In the provided link the above error message exists and is valid:

> unknown command 	404 	unknown command 	A [command](https://www.w3.org/TR/webdriver2/#dfn-commands) could not be executed because the [remote end](https://www.w3.org/TR/webdriver2/#dfn-remote-ends) is not aware of it.

**Comment by joerg1985** [comment]
Sorry i did not look too deep into this before opening, the encoding is correct.
The title of this issue should be: "status" could be removed from error responses.

e.g. the selenium server does respond in this case with:
```
{
  "value": {
    "stacktrace": "",
    "error": "unknown command",
    "message": "Unable to find handler for (GET) \u002fstatusdas"
  }
}
```
vs. the current response of appium: 
```
{
    "status": 9,
    "value": {
        "error": "unknown command",
        "message": "The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource",
        "stacktrace": ""
    }
}
```

**Comment by mykola-mokhnach** [comment]
yes, I assume these are historical leftovers from times where we were transitioning from jwp to w3c and had to basically support both protocols. Nevertheless, W3C does not deny having any additional fields defined there if the required payload is present and is formatted properly.

The cleanup should be done eventually...

**Comment by eglitise** [comment]
The status field has been removed in Appium 3.0.0.
